Dispatchfile

```diff
@@ -0,0 +1,87 @@
+#!mesosphere/dispatch-starlark:v0.6
+# vi:syntax=python
+#
+#
+
+load("github.com/mesosphere/dispatch-catalog/starlark/stable/git@0.0.7", "git_resource")
+load("github.com/mesosphere/dispatch-catalog/starlark/stable/pipeline@0.0.7", "tag")
+load("github.com/mesosphere/dispatch-catalog/starlark/stable/k8s@0.0.7", "secret_var")
+
+
+git = "src-git"
+git_resource(
+  git,
+  url="https://github.com/mesosphere/kubeaddons-kommander",
+  revision="$(context.git.commit)")
+
+gitops = "gitops-git"
+git_resource(
+  gitops,
+  url="https://github.com/mesosphere/konvoy-soak",
+  revision="master")
+
+
+task("update-soak", inputs=[gitops, git], steps=[
+  v1.Container(
+    name="fetch-master",
+    image="alpine/git:v2.24.3",
+    workingDir="/workspace/" + gitops,
+    args=[
+      "fetch",
+      "origin",
+      "master"
+    ]
+  ),
+  v1.Container(
+    name = "update-kommander-version",
+    image = "mikefarah/yq:3.3.4",
+    workingDir = "/workspace/{}/aws/".format(gitops),
+    command = ["/bin/sh"],
+    args = [
+      "-c",
+      "yq w --doc 1 --inplace cluster.yaml spec.addons[3].configVersion $(context.git.tag)"
+    ]
+  ),
+  v1.Container(
+    name = "update-gitops-repo",
+    image = "mesosphere/update-gitops-repo:1.1.1",
+    workingDir =  "/workspace/" + git,
+    command = ["/bin/sh", "-c"],
+    args = [
+      """
+      set -x
```

Comment:
It would be nice if you could add a ref or url to the script instead of adding it all here. It is hard to maintain and read.